openerp/models.py

```diff
@@ -5918,6 +5933,10 @@ def recompute(self):
                     recs.browse(ids)._write(dict(vals))
             # mark computed fields as done
             map(recs._recompute_done, fs)
+        # OpenUpgrade start, reset blacklists:
```

Comment:
@StefanRijnhart Actually not after every write, but after handling all the changes required by migrating one module. This takes care of your remark that the property will need to be reset in the post-migration, but without the need to do this manually. Otherwise the blacklist might still be active in other modules being migrated (or if the user actually logs into the migrated server to check the migration results).